.github/workflows/build.yml

```diff
@@ -796,6 +796,7 @@ jobs:
         if: ${{ ( github.event_name == 'push' && github.ref == 'refs/heads/master' ) || github.event.inputs.create_release == 'true' }}
         run: |
           Copy-Item LICENSE .\build\bin\Release\llama.cpp.txt
+          Copy-Item .\examples\run\linenoise.cpp\LICENSE .\build\bin\Release\linenoise.cpp.txt
```

Comment:
This will only affect the Windows releases, but the macOS and linux releases should also be updated.

Reply:
How about now?